ext/simplexml/tests/009b.phpt

```diff
@@ -28,8 +28,29 @@ object(SimpleXMLElement)#%d (3) {
     string(5) "elem1"
   }
   ["elem1"]=>
-  string(10) "Bla bla 1."
+  object(SimpleXMLElement)#%d (3) {
```

Comment:
Looks like the `Bla bla 1.` text got lost completely here?

Reply:
That is actually to be expected for mixed content; text nodes are dropped. That is already the case if these elements are `var_dump()`ed directly (in this case `var_dump($sxe->elem1)`; only children of the SimpleXMLElement are treated wrong without this patch.

Reply:
That seems pretty weird ... but it's SimpleXML so :man_shrugging: